src/main/java/smile/identity/core/WebApi.java

```diff
@@ -226,40 +183,31 @@ private void validateEnrollWithId(JSONArray images, JSONObject idInfo) throws Ex
         }
```

Comment:
In WebApi::validateEnrollWithId()
     if ((image_type_id == 1) || (image_type_id == 3) || (entered.equals("true"))) {
                counter = counter + 1;
            }
why do you increment counter when (entered.equals("true"). It is set once, and may be set incorrectly by the dev.. but counter can increment even if there are no images provided? and the validation succeeds